runtime/moonbase/src/lib.rs

```diff
@@ -1197,6 +1197,15 @@ impl session_keys_primitives::GetVrfInput<session_keys_primitives::VrfInput<Slot
 	}
 }
 
+pub struct ReserveAccount;
+impl Get<AccountId20> for ReserveAccount {
+	fn get() -> AccountId20 {
+		use pallet_evm::AddressMapping;
+		// harcoded precompile address
```

Comment:
precompile?

Reply:
Yes the Randomness precompile address. @crystalin this was discussed briefly, what address should we assign as the pallet address which holds all the deposits for this pallet?

Reply:
removed this in favor of const PalletId